File: xiajiwangluo_store/src/main/webapp/js/service/depotService.js
//服务层
app.service('depotService',function($http){

    //库存查询
    this.findPage=function (pageNum,pageSize,da_list){
        return $http.post('../depot/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize,da_list);
    }

});

File: xiajiwangluo_store/src/main/java/com/xiajiwangluo/service/DepotService.java
package com.xiajiwangluo.service;

import com.xiajiwangluo.bean.beangroup.DA_list;
import com.xiajiwangluo.bean.entity.PageResult;

public interface DepotService {

    /***
     * 分页
     * @param pageNum 当前页面
     * @param pageSize 记录数
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize, DA_list da_list);
}
